src/dependency.rs

```diff
@@ -88,6 +96,12 @@ impl Dependency {
     }
 }
 
+impl ReverseDependency {
+    pub fn encodable(self) -> EncodableDependency {
```

Comment:
Is the `download` argument in `Dependency.encodable` still necessary, since it says it's only needed when generating a reverse dependency, and this looks like it handles generating a reverse dependency then?